pkg/porter/parameters.go

```diff
@@ -81,17 +81,14 @@ func (p *Porter) PrintParameters(ctx context.Context, opts ListOptions) error {
 		tp := dtprinter.DateTimePrinter{
 			Now: func() time.Time { return now },
 		}
-
-		printParamRow :=
-			func(v interface{}) []string {
-				cr, ok := v.(DisplayParameterSet)
-				if !ok {
-					return nil
-				}
-				return []string{cr.Namespace, cr.Name, tp.Format(cr.Status.Modified)}
-			}
-		return printer.PrintTable(p.Out, params, printParamRow,
-			"NAMESPACE", "NAME", "MODIFIED")
+		paramsSets := [][]string{}
+		for _, ps := range params[0].Parameters {
```

Comment:
This is done because each install should have 1 parameter set but the function above returns a []DisplayParameterSet but it's length is one.  Tested by adding more params and it still held true to be 1 per install.

Reply:
If this is supposed to handle more than 1 install, I can change this to deal with that behavior.

Reply:
Changed to handle more than one install.